newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Fandangit'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '74cb518b00c659ee0308399cfed37c6df9a1595b',
```

Comment:
UMA!!! remove this config file immediately!!! You're gonna get that warning email!!! 😮